glider_dac/models/deployment.py

```diff
@@ -83,7 +83,7 @@ def dap(self):
             'user': slugify(self.username),
             'deployment': slugify(self.name)
         }
-        dap_url = u"http://%(host)s/thredds/dodsC/deployments/%(user)s/%(deployment)s/%(deployment)s.nc3.nc" % args
+        dap_url = "http://%(host)s/thredds/dodsC/deployments/%(user)s/%(deployment)s/%(deployment)s.nc3.nc" % args
```

Comment:
Do all these URLs (including the ones below) need to be https or is the ```host``` localhost?

Reply:
`http://localhost` is still valid in the event `host` is `localhost`, so this still should work regardless.

Reply:
Sorry for confusion, the question was whether we should use https over http